### DISCUSSION TITLE
I am not seeing Answer block to connect as the next step

### DISCUSSION THREAD

**Discussion by prashantabellad** [langgenius/dify#4843] [General]
I am not seeing Answer block to connect as the next step

![image](https://github.com/langgenius/dify/assets/10412309/49217c2a-405f-4921-9b2a-ee81a8ac0f8c)

I have configured ollama-llama3:8b model as can be seen below. Please help me resolve this

<img width="423" alt="Screenshot 2024-05-31 at 7 14 50 PM" src="https://github.com/langgenius/dify/assets/10412309/bd207f7e-845f-4a6d-9712-72c7ab6a51d2">

**Comment by patryk20120** [comment]
You are creating workflow. Workflow does not have `Answer` block  - it has `End` block, where you can specify what output variables you want to return.
